### ISSUE TITLE
Compatibility with Nahimic audio enhancement 

### ISSUE THREAD

**Issue by Dovencio** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#3642]
**OS:** Windows 10 x64 

**GPMDP Version:** 4.6.1

**Issue Descriptions:** When Nahimic is open only loud crackling noises can be heard 

**Steps to Reproduce:** 
1. Play a song with any Nahimic service or process running

If I want to listen to music while I have Nahimic installed GPMDP breaks my audio and starts playing loud crackling sounds. GPMDP sometimes would work with it but it seemingly would break and have trouble playing the next song and turn off the audio completely. This issue would occur if effects are turned off and/or Nahimic is closed.

**Comment by LesterCovax** [comment]
Do you have "Experimental support for multiple audio channels such as 5.1 audio" enabled in GPMDP?  Sounds like it's a buffer issue with the Nahimic software...at least that's the type of symptoms I encounter with buffer issues in other virtual audio devices.

**Comment by Dovencio** [comment]
> Do you have "Experimental support for multiple audio channels such as 5.1 audio" enabled in GPMDP? Sounds like it's a buffer issue with the Nahimic software...at least that's the type of symptoms I encounter with buffer issues in other virtual audio devices.

Yes, I do. That causes the issue 100% of the time. The reason I think that this is an issue with GPMDP is that other software such as games, other music players and websites don't have this issue. It only comes up when I am using that experimental audio.
